Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
+
+/* Add discount code message */
+"Add your discount code bellow:" = "Fügen Sie Ihren Rabatt Code unten hinzu:";
```

Comment:
"bellow"  has only one "l" in english:  "below" is correct. 

"Rabatt Code" would usually be made a compound word in geman  ->  "Rabattcode"  or "Gutscheincode" or "Coupon".